src/smc_sagews/smc_sagews/sage_salvus.py

```diff
@@ -2763,11 +2748,34 @@ def plot3d_using_matplotlib(expr,
     plt.show()
 
 
-from sage.plot.graphics import Graphics, GraphicsArray
+# 8.9 introduced https://doc.sagemath.org/html/en/reference/plotting/sage/plot/multigraphics.html#sage.plot.multigraphics.MultiGraphics
+try:
```

Comment:
You should do the logic in the opposite order or actually check the version of Sage.  The way you are doing it now you are assuming ` from sage.plot.graphics import GraphicsArray` fails implies "pre 8.9".  However, in two years for sage-9.4 somebody could happily create a module called graphics that exports GraphicsArray, and suddenly this code totally breaks!

In fact, that's not such a crazy scenario.  Imagine this happening:
1. People complain that sage.plot.graphics.GraphicsArray went away and lots of code broke as a result (the public api of sage changed without a deprecation warning),
2. Somebody decides to add back a shim so that sage.plot.graphics.GraphicsArray exists again.
3. Boom -- your code just broke.